src/seedu/addressbook/ui/TextUi.java

```diff
@@ -8,10 +8,7 @@
 
 import java.io.InputStream;
 import java.io.PrintStream;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Optional;
-import java.util.Scanner;
+import java.util.*;
```

Comment:
Why do you import java.util.* instead of only the required packages?

Reply:
Thanks for catching that. It would seem that IntelliJ automatically added a java.util.* import.